test/mapKeys.js

```diff
@@ -0,0 +1,9 @@
+var assert = require('assert');
+
+var R = require('..');
+
+describe('mapKeys', function() {
+  it('returns an object with each of the keys passed set to value returned for it by the given function', function() {
+    assert.deepEqual(R.mapKeys(R.toUpper, ['a', 'b', 'c']), {a: 'A', b: 'B', c: 'C'});
+  });
+});
```

Comment:
definitely needs more tests. what happens for empty list? is the function curried? what if the input list is a multiset? which one wins?